paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/CustomerSheetTest.kt

```diff
@@ -41,8 +42,9 @@ internal class CustomerSheetTest {
     lateinit var integrationType: IntegrationType
 
     @Test
-    fun testSuccessfulCardSave() = activityScenarioRule.runCustomerSheetTest(
+    fun testSuccessfulCardSaveAndAttachToCustomer() = activityScenarioRule.runCustomerSheetTest(
         integrationType = integrationType,
+        createAdapter = createTestAdapter(useSetupIntent = false),
```

Comment:
I'm tempted to see if we could build this into our parameterization of tests. Are there cases where we can't do that?